### PR TITLE
feat: update API7 Whitepaper(EN) and API7 Performance Report(ZH)

### PR DESCRIPTION
**Pull request by guoqqqi**
1. update API7 Whitepaper(EN) pdf
2. update API7 Performance Report(ZH) pdf

### CODE REVIEW

**Comment by yzeng25** [comment]
PDFs lgtm 👍

File: components/Footer/_data.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const EN_US_links: LinkGroupData[] = [
         href: "/resources/apisix-devcon-2020"
       }, , {
         label: "API7 Whitepaper",
-        href: "https://static.apiseven.com/202108/API7-WhitePaper-EN.pdf"
+        href: "https://static.apiseven.com/202108/API7-WhitePaper-EN(1).pdf"
```
Comment (Baoyuantop) [inline components/Footer/_data.tsx:49]: How about replacing the file directly?

Comment (guoqqqi) [reply components/Footer/_data.tsx:49]: I don't have permission to delete the old file, the replacement link is thrown away as the previous link

Comment (Baoyuantop) [reply components/Footer/_data.tsx:49]: I just think it's not very elegant

**Comment by guoqqqi** [comment]
Temporarily removed, API7 Review Report

**Comment by guoqqqi** [comment]
cc @Baoyuantop

**Review by Baoyuantop — APPROVED** [review]

**Pull request merged**